### ISSUE TITLE
[BUG] scipy not including the `kulsinski` or `kulczynski1`

### ISSUE THREAD

**Issue by direkkakkar319-ops** [alan-turing-institute/sktime#10190]
**Describe the bug**
in file `test_scipy_dist.py` has a helper fun `get_kul_name()` it imports the `kulczynski1`. 
if the import failes then it goes to `kulsinski` but they both are not there in the new version of `scipy` that i `1.17`

due to this there would be import and value errors raised as found when i ran the tests in the main
<img width="1919" height="367" alt="Image" src="https://github.com/user-attachments/assets/4853f247-5788-482b-a710-9f131a3e45f1" />

**To Reproduce**
run the scipy test with scipy 1.17.1 installed:
```python
python -m pytest sktime/dists_kernels/tests/test_scipy_dist.py::test_scipydist
```

**Expected behavior**
test should pass under scipy 1.17 by excluding removed metrics such as `kulsinski`, `kulczynski1`,

**Versions**
<details>
System:
    python: 3.12.3 (main, Mar 23 2026, 19:04:32) [GCC 13.3.0]
executable: /mnt/f/sktime/venv/bin/python
   machine: Linux-6.6.87.2-microsoft-standard-WSL2-x86_64-with-glibc2.39

Python dependencies:
          pip: 24.0
       sktime: 0.40.1
      sklearn: 1.7.2
       skbase: 0.13.1
        numpy: 2.3.5
        scipy: 1.17.1
       pandas: 2.3.3
   matplotlib: None
       joblib: 1.5.3
huggingface-hub: None
        numba: None
     pmdarima: None
pytorch-forecasting: None
   skforecast: None
        skpro: 2.12.0
statsforecast: None
  statsmodels: None
 transformers: None
      tsfresh: None
      tslearn: None
        torch: None
   tensorflow: None
(venv) direk@dkakkar:/mnt/f/sktime$ 
</details>

**Comment by direkkakkar319-ops** [comment]
Hi @fkiraly @marrov i am thinking of just to add a fall back function that will deal with the import problem and may be we can skip metric in it ?? is it the correct soultion ??
